index.js

```diff
@@ -571,12 +571,8 @@ function buildArray (context, location) {
     `
   }
 
-  if (largeArrayMechanism !== 'default') {
-    if (largeArrayMechanism === 'json-stringify') {
-      functionCode += `if (arrayLength && arrayLength >= ${largeArraySize}) return JSON.stringify(obj)\n`
-    } else {
```

Comment:
because `validLargeArrayMechanisms` -> this throw will never happen (we get an early throw if `largeArrayMechanism` is not `default` or  `json-stringify`) so no need for it

Reply:
thats true